forloop_modules/function_handlers/variable_handlers.py

```diff
@@ -17,6 +17,8 @@
 from forloop_modules.function_handlers.auxilliary.abstract_function_handler import AbstractFunctionHandler, Input
 from forloop_modules.errors.errors import CriticalPipelineError
 
+from forloop_modules.function_handlers.interactive_handlers import ConsoleEntryHandler #Console input
```

Comment:
This import seems on wrong place, rather add all interactive_handlers as an import in "pipeline_function_handlers" file

Reply:
Or move ConsoleEntryHandler in variable_handlers.py

Reply:
The structure is:
pipeline_function_handlers -> imports cca 15 independent files with various handlers
it shouldnt be nested more than 1 level